### PR TITLE
refactor dockerfile

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: yeya24 <yb532204897@gmail.com>
Simply refactor the Dockerfiles of amNotifier and commentMonitor

### CODE REVIEW

**Comment by krasi-georgiev** [comment]
@geekodour I will let you review this one.

**Comment by geekodour** [comment]
LGTM. 

@yeya24  Would you mind updating [fake-webserver](https://github.com/prometheus/prombench/tree/master/tools/fake-webserver) and [scaler](https://github.com/prometheus/prombench/tree/master/tools/scaler) and [prombench](https://github.com/prometheus/prombench/blob/master/prombench/cmd/prombench/Dockerfile) to use multi staged builds like amGithubNotifier and comment-monitor along with this PR.

We should be able to add automated docker image builds after that and continue the work at #218

File: tools/commentMonitor/Dockerfile [hunk 1]
```diff
@@ -4,18 +4,11 @@ FROM golang:1.12-alpine AS builder
...
-COPY tools/commentMonitor/ghclient.go .
-
-COPY go.mod .
-COPY go.sum .
-RUN go mod download
+COPY tools/commentMonitor/*.go ./
+COPY go.* ./
 
-RUN CGO_ENABLED=0 GOOS=linux go build -o /go/bin/commentMonitor
+RUN GO111MODULE=on CGO_ENABLED=0 GOOS=linux go build -o /go/bin/commentMonitor
```
Comment (geekodour) [inline tools/commentMonitor/Dockerfile]: `CGO_ENABLED=0` can be replaced with `-tags netgo` to keep things consistent among the Dockerfiles, It should work the same for this.

**Comment by yeya24** [comment]
> LGTM.
> 
> @yeya24 Would you mind updating [fake-webserver](https://github.com/prometheus/prombench/tree/master/tools/fake-webserver) and [scaler](https://github.com/prometheus/prombench/tree/master/tools/scaler) and [prombench](https://github.com/prometheus/prombench/blob/master/prombench/cmd/prombench/Dockerfile) to use multi staged builds like amGithubNotifier and comment-monitor along with this PR.
> 
> We should be able to add automated docker image builds after that and continue the work at #218

Sure, I will pin you when I finish it.

**Comment by krasi-georgiev** [comment]
@yeya24 would you mind updateding that small nit from @geekodour and I will merge it.

**Comment by yeya24** [comment]
@geekodour I will make other Dockerfiles multi-stage in another PR as @krasi-georgiev said. Could you please review this again?

**Comment by geekodour** [comment]
LGTM

cc @krasi-georgiev

**Pull request merged**